src/HonzaBotner.Discord.Services/Commands/MessageCommands.cs

```diff
@@ -219,5 +221,75 @@ public async Task Dump(CommandContext ctx, [Description("URL of the message.")]
                 await ctx.Message.RespondAsync($"TODO: {roles.Count}");
             }*/
         }
+
+        [Group("buttons")]
+        [Description("Module used to edit button interactions on messages")]
+        [ModuleLifespan(ModuleLifespan.Transient)]
+        public class ButtonCommands : BaseCommandModule
+        {
+            private readonly IButtonManager _buttonManager;
+
+            public ButtonCommands(IButtonManager manager)
+            {
+                _buttonManager = manager;
+            }
+
+            [Description("Deletes all button interactions on the message")]
+            [Command("remove")]
+            public async Task RemoveButtons(CommandContext ctx, [Description("URL of the message")] string url)
+            {
+                DiscordMessage? message = await DiscordHelper.FindMessageFromLink(ctx.Guild, url);
+                if (message == null)
+                {
+                    throw new ArgumentOutOfRangeException($"Couldn't find message with link: {url}");
+                }
+
+                DiscordEmoji reactEmoji = DiscordEmoji.FromName(ctx.Client, ":+1:");
+                try
+                {
+                    await _buttonManager.RemoveButtonsFromMessage(message);
+                }
+                catch (Exception exception)
+                {
+                    if (exception is not UnauthorizedException)
+                    {
+                        throw;
+                    }
```

Comment:
This syntax is unnecessary. Just catch UnauthorizedException

Reply:
Sry, it's leftover.. I was catching more exceptions in one block and later removed most of them..